extension.js

```diff
@@ -363,31 +362,33 @@ class MprisLabel extends PanelMenu.Button {
 	}
 
 	_refresh() {
-		const REFRESH_RATE = this.settings.get_int('refresh-rate');
+		const refreshRate = this.settings.get_int('refresh-rate');
 
 		let prevPlayer = this.player;
 
 		try {
 			this.players.updateFilterList();
 			this.players.updateActiveList();
 
-			if(this.panelControls && this.player != null | undefined)
-				this.panelControls.updateControls(this.player);
+			if (this.panelControls && this.player != null | undefined) {
+				this.panelControls.updateControls(this.player)
+			}
 		}
-		catch {
-			; //do nothing
+		catch(err){
+			log("Mpris _refresh: " + err);
 		}
 
 		this.player = this.players.pick();
 
-		if(this.player != prevPlayer)
+		if(this.player != null | undefined && this.player != prevPlayer) {
```

Comment:
We don't use `{ }` in single line `if` statements. Again, just following the original extension style.